upstart/security-hq.conf

```diff
@@ -0,0 +1,32 @@
+# upstart script for security-hq
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+setuid securityhq
+chdir /security-hq
+
+script
+  IF_64_BIT_OPTION=""
+  if $(java -version 2>&1 | grep -q 64-Bit);
+  then
+    IF_64_BIT_OPTIONS="-J-XX:+UseCompressedOops -J-XX:+UseConcMarkSweepGC"
+  fi
+
+  # Incremental mode if we are on 1 or 2 CPUs http://www.oracle.com/technetwork/java/javase/gc-tuning-6-140523.html#icms
+  INCREMENTAL_MODE=""
```

Comment:
this seems to be deprecated in java 8
see https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/cms.html

`Note that the incremental mode is being deprecated in Java SE 8 and may be removed in a future major release`